CA_DataUploaderLib/IOconf/IOconfRow.cs

```diff
@@ -12,12 +12,14 @@ public IOconfRow(string row, int lineNum, string type)
             Type = type;
             LineNumber = lineNum;
             var list = ToList();
-            if (list[0] != Type) throw new Exception("IOconfRow: wrong format: " + Row);
+            if (list[0] != Type && type != "Unknown") throw new Exception("IOconfRow: wrong format: " + Row);
```

Comment:
If we go with this, we should probably still reject running with these somewhere else.

reason: it turns mistypes into unknown rows without letting the user know.

ps. without the specific type loaders we don't know how many values they will expand to anyway